src/System.Private.CoreLib/shared/System/Diagnostics/Tracing/StubEnvironment.cs

```diff
@@ -364,6 +366,44 @@ class PermissionSetAttribute : Attribute
         public bool Unrestricted { get; set; }
     }
 }
+
+namespace System.Security
+{
+    public enum SecurityAction
+    {
+        // Demand permission of all caller
+        Demand = 2,
+
```

Comment:
Do you need all these values to make the build work? Wouldn't it be better to just ifdef out the usage for CoreCLR and CoreRT?